features/XouBaseApiLoginAndRegister/Repo/UserModel.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UserModel.swift
+//  XouDevSpec
+//
+//  Created by Xander Schoeman on 2020/04/19.
+//
+
+import Foundation
+
+public struct User: Decodable {
+    public var id: Int = 0
+    public  var username: String = ""
+    public var password: String = ""
+    public var profileImage: String? = ""
+    public var fullName: String = ""
+    public var email: String = ""
+    public var gender: String? = ""
+    public var age: Int = 0
+    public var favouriteCount: Int = 0
+    public var genrePref: String = ""
+
+    public init() {        
+    }
```

Comment:
won't need to specify this init either, a default one will be provided. If any of the variables (non-optionals) are given default values it will not include those in the default init.

Reply:
So funny thing, without it I cannot access the struct at all since it is a development pod. Even though I entirely agree the init is not needed, but without it is gives be an accessibility error due to protection level.

Reply:
True, that makes sense. I didn't think of it being used outside of the project. Thanks.